.github/workflows/ci.yml

```diff
@@ -35,9 +35,12 @@ jobs:
           CHANGED=""
           MODULES_ARG=""
 
+          # If changed file have some special character, its path is surrounded with quotes which causing the if statement fail
+          CHANGED_FILE=$(echo ${{ steps.files.outputs.all_changed_and_modified_files }} | sed 's/\"/\\"/')
```

Comment:
I guess this is a specific solution to avoid special characters to attempt to escape the quotes, but still valid for our purpose.